### PR TITLE
support onTimer handler in useConnectCall

### PR DESCRIPTION
**Pull request by wittejm**
Video handler emits a timer event here: https://github.com/Ameelio/connect-video-handler/blob/1306d2db8697c64600cf77b7d36567ee83b3424f/src/Room.ts#L416

### CODE REVIEW

File: src/useConnectCall.ts [hunk 1]
```diff
@@ -30,6 +30,7 @@ type Props = {
   authInfo: Participant & { token: string };
   onPeerConnected?: (user: Participant) => void;
   onPeerDisconnected?: (user: Participant) => void;
+  onTimer?: (name: "maxDuration", msRemaining: number) => void;
```
Comment (cainlevy) [inline src/useConnectCall.ts:33]: i'm curious what you discovered the host app can do with `onTimer`. some kind of setInterval+setState solution, maybe?

**Review by cainlevy — APPROVED** [review]

File: src/useConnectCall.ts (hunk 1 above)
Comment (wittejm) [reply src/useConnectCall.ts:33]: I didn’t yet. My thinking about usage in the app went only as far as seeing where a handler would live and pass to useConnectCall. Also, I actually don’t mean to merge this PR until I try using it in pathways. (Just wanted a sanity check!)

Comment (cainlevy) [reply src/useConnectCall.ts:33]: looks sane to me.

i imagine that weaving this event through each layer probably felt a bit ceremonial? it's possible that when we discover what to do with the timer event we bring some of that functionality into here and it gets a bit more interesting.

Comment (wittejm) [reply src/useConnectCall.ts:33]: I see. Yeah, it took me a while to understand the layers of emitters&handlers but now that I've done it once it does all feel quite basic :) . I'll keep in mind the idea of building a time event emitter to suit client usage as necessary.

**Comment by wittejm** [comment]
Merging this because I'm stopping this vector of work, but this wrapper at least matches the current form of the videoHandler event

**Pull request merged**